common/authentication/postgresql/metadata.go

```diff
@@ -19,11 +19,15 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/internal/exported"
```

Comment:
Note that you're trying to import an `internal` package, which cannot be imported (that's just a limitation in Go)

Reply:
yeah i know we cant pull from internal. I think my IDE messed up by not telling me the err here lol